src/h5web/metadata-viewer/EntityInfo.tsx

```diff
@@ -24,17 +24,21 @@ function EntityInfo(props: Props): ReactElement {
         </tr>
       </thead>
       <tbody>
-        {'type' in entity && (
+        {(isDataset(entity) || isDatatype(entity)) && (
           <tr>
             <th scope="row">Type</th>
-            <td>{isBaseType(entity.type) ? entity.type.class : 'Complex'}</td>
+            <td>
+              {typeof entity.type === 'string'
```

Comment:
This condition puzzles me.

`entity.type` can be `string` or something that has a `class` ?

Reply:
Yeah, it can be the ID of a datatype entity.